provisioning_client/src/prov_device_ll_client.c

```diff
@@ -1118,29 +1118,43 @@ void Prov_Device_LL_DoWork(PROV_DEVICE_LL_HANDLE handle)
         }
         if (prov_info->is_connected || prov_info->prov_state == CLIENT_STATE_ERROR)
         {
+            tickcounter_ms_t current_time = 0;
```

Comment:
Shouldn't we get the real time every time?

Reply:
it looks like it is also used in CLIENT_STATE_STATUS_SEND... so it will be zero?

Reply:
No, I had to move it here because if you use the same var name in different switch cases the compiler (at least VS2019) will fail. We only get the time in 2 switch cases.